composer.json

```diff
@@ -10,7 +13,8 @@
         "psr/http-message": "~1.0",
         "psr/http-server-handler" : "*",
         "psr/http-server-middleware" : "*",
-        "zendframework/zend-diactoros": "^1.6"
+        "zendframework/zend-diactoros": "^1.6",
+        "loris/instrumentmanager" : "dev-master"
```

Comment:
Should this be `loris2` as per your PR description? Could fix the Travis error also.

Reply:
My understanding is that it needs to match what's in the composer.json of the repo, and it's just "loris" there: https://github.com/loris2/instrumentmanager/blob/master/composer.json#L2

Reply:
Hmmm maybe Travis si having an issue with this line then?

Reply:
..though I just noticed the URL for "loris" is wrong in the other repo's composer.json, so I'll try fixing that and see if it helps

Reply:
The "vcs" line tells composer that it should look for packages in that repo, otherwise composer will just use packagist.org. (If this works we should probably make packagist releases of LORIS and its modules, but until we're committed to adding modules with "composer require" it's probably not worth it.)

Reply:
Thanks for the explanation!

Reply:
it was that line that was the problem.. now the error is more like what I expected since there's no release with this change:

```
composer install
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Your requirements could not be resolved to an installable set of packages.
  Problem 1
    - Installation request for loris/instrumentmanager dev-master -> satisfiable by loris/instrumentmanager[dev-master].
    - loris/instrumentmanager dev-master requires aces/loris >20.0 -> no matching package found.
```

Reply:
I'm not sure how I can bootstrap this before it's merged.. I can't have the instrumentmanager require the major branch either, because it's also not merged into major.

Reply:
Does it rely on the code changes below and that's why it needs 21? Is it possible those changes could go into 20.x? 

Or maybe this comes out in 21.1 after the below is merged. Essentially merge the underlying code without including the test example.

I've got a hazy understanding of this but... don't see how you get out of this dependency circle without delaying some piece of it.

Reply:
Yeah, these are the code changes that I had to do in order to get it to load when using a local file:// url for the VCS line.

Reply:
I'll try eliminating the circular dependency by not declaring the loris dependency on the other side until there's a release which includes the changes